raiden_libs/network/matrix/client.py

```diff
@@ -388,6 +399,9 @@ def set_account_data(self, type_: str, content: Dict[str, Any]) -> dict:
     def set_post_sync_hook(self, hook: Callable[[str], None]):
         self._post_hook_func = hook
 
+    def set_sync_token(self, sync_token: str) -> None:
+        self.sync_token = sync_token
```

Comment:
thanks. I guess you can now also turn it into `_sync_token`

Reply:
No, it's inherited from the base class.